cypress/support/pages/management/pageManagement.js

```diff
@@ -0,0 +1,158 @@
+  static #firstNameEditForm = "input[name='firstName']"
+  static #lastNameEditForm = "input[name='lastName']"
+  static #emailEditForm = "input[name='email']"
+  static #submitButton = '[data-testid="button_save_user"]'
+  static #deleteButton = '[data-testid="button_delete_user"]'
+  static #cancelButtonConfirmation = '[data-testid="btn-cancel"]'
+  static #deleteButtonCancel = '.cancel-modal button.historyback'
+  static #deleteButtonConfirmation = '[data-testid="btn-submit"]'
+  static #newSchoolAdminPageButton = '[data-testid="button_new_admin_page"]'
+  static #chatToggleSwitch = '.rocketchat-switch'
+  static #videoconferenceToggleSwitch = '.videoconference-switch'
+  static #saveGeneralSettingsButton = '.my-5'
+  static #tableContents = '[data-testid="table-data-body"]'
+
+  clickOnFAB() {
+    cy.get(Management.#fabButton).click()
+    cy.get(Management.#createUserButton).click()
+  }
+
+  fillStudentCreationForm() {
+    cy.get(Management.#firstNameCreationForm).type('Adam')
+    cy.get(Management.#lastNameCreationForm).type('Riese')
+    cy.get(Management.#emailCreationForm).type('adam.riese@example.com')
+  }
+
+  fillTeacherCreationForm() {
+    cy.get(Management.#firstNameCreationForm).type('Adam')
+    cy.get(Management.#lastNameCreationForm).type('Rose')
+    cy.get(Management.#emailCreationForm).type('adam.rose@example.com')
+  }
+
+  clickOnAddButton() {
+    cy.get(Management.#addButton).click()
+  }
+
+  enterNameForSearch() {
+    cy.get(Management.#searchbar).type('Adam')
+  }
+
+  clickEditStudentButton() {
+    cy.get(Management.#editStudentButton).eq(0).click()
+  }
+
+  clickEditTeacherButton() {
+    cy.get(Management.#editTeacherButton).eq(0).click()
+  }
+
+  changeStudentUserInformation() {
+    cy.get(Management.#firstNameEditForm).clear()
+    cy.get(Management.#firstNameEditForm).type('Alex')
+    cy.get(Management.#lastNameEditForm).clear()
+    cy.get(Management.#lastNameEditForm).type('Abramovic')
+    cy.get(Management.#emailEditForm).clear()
+    cy.get(Management.#emailEditForm).type('alex.abramovic@example')
+  }
+
+  changeTeacherUserInformation() {
+    cy.get(Management.#firstNameEditForm).clear()
+    cy.get(Management.#firstNameEditForm).type('Amber')
+    cy.get(Management.#lastNameEditForm).clear()
+    cy.get(Management.#lastNameEditForm).type('Adams Young')
+    cy.get(Management.#emailEditForm).clear()
+    cy.get(Management.#emailEditForm).type('amber.adams-young@example')
+  }
+
+  clickSaveButton() {
+    cy.get(Management.#submitButton).eq(0).click()
+  }
+
+  clickDeleteButton() {
+    cy.get(Management.#deleteButton).click()
+  }
+
+  clickDeleteButtonInPopup() {
+    cy.get(Management.#deleteButtonConfirmation).click({ multiple: true, force:true})
+  }
+
+  clickNewAdminPageButton() {
+    cy.get(Management.#newSchoolAdminPageButton).click()
+    cy.url().should('include', '/administration/school-settings')
+  }
+
+  clickVideoconferenceToggleSwitch() {
```

Comment:
`clickVideoConferenceToggleSwitch`

Reply:
this is already in main, but I updated it in this PR.